examples/EnableServiceProfilerForAzureFunction/Program.cs

```diff
@@ -0,0 +1,31 @@
+// -----------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+// -----------------------------------------------------------------------------
+
+using Microsoft.ApplicationInsights.Profiler.Core.Contracts;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
+    .ConfigureLogging(logging =>
+    {
+        logging.AddConsole();
+    })
+    .ConfigureServices(services => {
+        services.AddApplicationInsightsTelemetryWorkerService();
+        services.AddServiceProfiler(p =>
+        {
+            p.Duration = TimeSpan.FromSeconds(10);
```

Comment:
These values might be good if you know what you are doing. As an general example, do you mind set them to normal values?